components/tidb_query/src/expr/builtin_string.rs

```diff
@@ -1,6 +1,7 @@
 // Copyright 2018 TiKV Project Authors. Licensed under Apache-2.0.
 
 use base64;
+use bstr;
```

Comment:
This `use` statement can be removed.

Reply:
It can be removed. But there are many statements like this, I not sure whether it is a coding style to make it easy to find what crate this file use so I add it.
So is it just an accidentally introduced style? If it is, I will remove it ( and recommend removing them completely to prevent others following this style)

Reply:
In Rust 2018 it is no longer needed so we normally don't introduce new ones.

Reply:
Removed